### ISSUE TITLE
Problems "go getting" buffalo

### ISSUE THREAD

**Issue by hubyhuby** [gobuffalo/buffalo#899]
Hi there,
i am trying to simply go get buffalo (on Ubuntu 17.10).
Tried this one several times: 
go get -u -v -tags nosqlite github.com/gobuffalo/buffalo/buffalo

I got some kind of conflicts with lots of previously go getted libraries...
So I finally deleted my src folder completely! Started fresh;)
But no luck after all the log below, the buffalo command is not intalled :(
buffalo new coke
buffalo: command not found

PS: is there a forum for buffalo ?

LOG of the go get :

go get -u -v -tags nosqlite github.com/gobuffalo/buffalo/buffalo
github.com/gobuffalo/buffalo (download)
github.com/fatih/color (download)
github.com/gobuffalo/envy (download)
github.com/joho/godotenv (download)
github.com/mitchellh/go-homedir (download)
github.com/gobuffalo/makr (download)
github.com/markbates/inflect (download)
github.com/pkg/errors (download)
github.com/sirupsen/logrus (download)
Fetching https://golang.org/x/crypto/ssh/terminal?go-get=1
Parsing meta tags from https://golang.org/x/crypto/ssh/terminal?go-get=1 (status code 200)
get "golang.org/x/crypto/ssh/terminal": found meta tag main.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/ssh/terminal?go-get=1
get "golang.org/x/crypto/ssh/terminal": verifying non-authoritative meta tag
Fetching https://golang.org/x/crypto?go-get=1
Parsing meta tags from https://golang.org/x/crypto?go-get=1 (status code 200)
golang.org/x/crypto (download)
Fetching https://golang.org/x/sys/unix?go-get=1
Parsing meta tags from https://golang.org/x/sys/unix?go-get=1 (status code 200)
get "golang.org/x/sys/unix": found meta tag main.metaImport{Prefix:"golang.org/x/sys", VCS:"git", RepoRoot:"https://go.googlesource.com/sys"} at https://golang.org/x/sys/unix?go-get=1
get "golang.org/x/sys/unix": verifying non-authoritative meta tag
Fetching https://golang.org/x/sys?go-get=1
Parsing meta tags from https://golang.org/x/sys?go-get=1 (status code 200)
golang.org/x/sys (download)
github.com/gobuffalo/packr (download)
Fetching https://golang.org/x/sync/errgroup?go-get=1
Parsing meta tags from https://golang.org/x/sync/errgroup?go-get=1 (status code 200)
get "golang.org/x/sync/errgroup": found meta tag main.metaImport{Prefix:"golang.org/x/sync", VCS:"git", RepoRoot:"https://go.googlesource.com/sync"} at https://golang.org/x/sync/errgroup?go-get=1
get "golang.org/x/sync/errgroup": verifying non-authoritative meta tag
Fetching https://golang.org/x/sync?go-get=1
Parsing meta tags from https://golang.org/x/sync?go-get=1 (status code 200)
golang.org/x/sync (download)
Fetching https://golang.org/x/net/context?go-get=1
Parsing meta tags from https://golang.org/x/net/context?go-get=1 (status code 200)
get "golang.org/x/net/context": found meta tag main.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/context?go-get=1
get "golang.org/x/net/context": verifying non-authoritative meta tag
Fetching https://golang.org/x/net?go-get=1
Parsing meta tags from https://golang.org/x/net?go-get=1 (status code 200)
golang.org/x/net (download)
github.com/gobuffalo/plush (download)
github.com/gobuffalo/tags (download)
github.com/fatih/structs (download)
github.com/markbates/validate (download)
github.com/markbates/going (download)
github.com/satori/go.uuid (download)
github.com/serenize/snaker (download)
github.com/russross/blackfriday (download)
github.com/shurcooL/github_flavored_markdown (download)
github.com/microcosm-cc/bluemonday (download)
Fetching https://golang.org/x/net/html?go-get=1
Parsing meta tags from https://golang.org/x/net/html?go-get=1 (status code 200)
get "golang.org/x/net/html": found meta tag main.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/html?go-get=1
get "golang.org/x/net/html": verifying non-authoritative meta tag
Fetching https://golang.org/x/net/html/atom?go-get=1
Parsing meta tags from https://golang.org/x/net/html/atom?go-get=1 (status code 200)
get "golang.org/x/net/html/atom": found meta tag main.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/html/atom?go-get=1
get "golang.org/x/net/html/atom": verifying non-authoritative meta tag
github.com/shurcooL/highlight_diff (download)
github.com/sergi/go-diff (download)
github.com/sourcegraph/annotate (download)
github.com/sourcegraph/syntaxhighlight (download)
github.com/shurcooL/highlight_go (download)
github.com/shurcooL/octiconssvg (download)
github.com/shurcooL/sanitized_anchor_name (download)
github.com/spf13/cobra (download)
github.com/spf13/pflag (download)
github.com/markbates/pop (download)
github.com/cockroachdb/cockroach-go (download)
github.com/lib/pq (download)
github.com/go-sql-driver/mysql (download)
github.com/jmoiron/sqlx (download)
github.com/mattn/anko (download)
github.com/daviddengcn/go-colortext (download)
Fetching https://gopkg.in/yaml.v2?go-get=1
Parsing meta tags from https://gopkg.in/yaml.v2?go-get=1 (status code 200)
get "gopkg.in/yaml.v2": found meta tag main.metaImport{Prefix:"gopkg.in/yaml.v2", VCS:"git", RepoRoot:"https://gopkg.in/yaml.v2"} at https://gopkg.in/yaml.v2?go-get=1
gopkg.in/yaml.v2 (download)
github.com/markbates/deplist (download)
github.com/markbates/grift (download)
github.com/markbates/refresh (download)
github.com/fsnotify/fsnotify (download)
github.com/markbates/sigtx (download)
github.com/fatih/color/vendor/github.com/mattn/go-colorable
github.com/fatih/color/vendor/github.com/mattn/go-isatty
github.com/gobuffalo/buffalo/generators/assets
github.com/joho/godotenv
github.com/mitchellh/go-homedir
github.com/pkg/errors
github.com/fatih/color
golang.org/x/sys/unix
github.com/gobuffalo/envy
golang.org/x/net/context
golang.org/x/sync/errgroup
github.com/gobuffalo/buffalo/generators
github.com/markbates/inflect
github.com/gobuffalo/packr
github.com/gobuffalo/packr/builder
github.com/gobuffalo/plush/token
github.com/gobuffalo/plush/ast
github.com/gobuffalo/plush/lexer
github.com/fatih/structs
github.com/gobuffalo/plush/parser
github.com/gobuffalo/tags
golang.org/x/crypto/ssh/terminal
github.com/markbates/going/wait
github.com/markbates/validate
github.com/sirupsen/logrus
github.com/satori/go.uuid
github.com/markbates/going/defaults
github.com/serenize/snaker
github.com/russross/blackfriday
github.com/markbates/validate/validators
golang.org/x/net/html/atom
golang.org/x/net/html
github.com/sergi/go-diff/diffmatchpatch
github.com/gobuffalo/makr
github.com/gobuffalo/buffalo/meta
github.com/gobuffalo/tags/form
github.com/sourcegraph/annotate
github.com/sourcegraph/syntaxhighlight
github.com/gobuffalo/buffalo/generators/assets/standard
github.com/gobuffalo/buffalo/generators/assets/webpack
github.com/shurcooL/highlight_diff
github.com/shurcooL/highlight_go
github.com/gobuffalo/tags/form/bootstrap
github.com/shurcooL/sanitized_anchor_name
github.com/spf13/pflag
github.com/gobuffalo/buffalo/generators/action
github.com/gobuffalo/buffalo/generators/docker
github.com/gobuffalo/buffalo/generators/grift
github.com/gobuffalo/buffalo/generators/mail
github.com/microcosm-cc/bluemonday
github.com/shurcooL/octiconssvg
github.com/gobuffalo/buffalo/generators/resource
github.com/gobuffalo/buffalo/generators/refresh
github.com/lib/pq/oid
github.com/lib/pq
github.com/go-sql-driver/mysql
# github.com/lib/pq
lib/pq/conn_go18.go:15: undefined: driver.NamedValue
lib/pq/conn_go18.go:33: undefined: driver.NamedValue
lib/pq/conn_go18.go:47: undefined: driver.TxOptions
lib/pq/conn_go18.go:50: undefined: sql.IsolationLevel
lib/pq/conn_go18.go:51: undefined: sql.LevelDefault
lib/pq/conn_go18.go:53: undefined: sql.LevelReadUncommitted
lib/pq/conn_go18.go:55: undefined: sql.LevelReadCommitted
lib/pq/conn_go18.go:57: undefined: sql.LevelRepeatableRead
lib/pq/conn_go18.go:59: undefined: sql.LevelSerializable
github.com/jmoiron/sqlx/reflectx
github.com/jmoiron/sqlx
github.com/spf13/cobra
# github.com/go-sql-driver/mysql
go-sql-driver/mysql/connection_go18.go:42: undefined: driver.TxOptions
go-sql-driver/mysql/connection_go18.go:48: undefined: sql.IsolationLevel
go-sql-driver/mysql/connection_go18.go:48: undefined: sql.LevelDefault
go-sql-driver/mysql/connection_go18.go:62: undefined: driver.NamedValue
go-sql-driver/mysql/connection_go18.go:81: undefined: driver.NamedValue
go-sql-driver/mysql/connection_go18.go:115: undefined: driver.NamedValue
go-sql-driver/mysql/connection_go18.go:134: undefined: driver.NamedValue
go-sql-driver/mysql/connection_go18.go:199: undefined: driver.NamedValue
go-sql-driver/mysql/utils_go18.go:24: undefined: driver.NamedValue
go-sql-driver/mysql/utils_go18.go:36: undefined: driver.IsolationLevel
go-sql-driver/mysql/connection_go18.go:48: too many errors
github.com/markbates/going/randx
# github.com/jmoiron/sqlx
jmoiron/sqlx/named_context.go:41: n.Stmt.ExecContext undefined (type *Stmt has no field or method ExecContext)
jmoiron/sqlx/named_context.go:41: not enough arguments to return
jmoiron/sqlx/named_context.go:51: n.Stmt.QueryContext undefined (type *Stmt has no field or method QueryContext)
jmoiron/sqlx/named_context.go:51: not enough arguments to return
jmoiron/sqlx/sqlx_context.go:20: db.PingContext undefined (type *DB has no field or method PingContext)
jmoiron/sqlx/sqlx_context.go:122: cannot use db (type *DB) as type namedPreparerContext in argument to prepareNamedContext:
	*DB does not implement namedPreparerContext (missing PrepareContext method)
jmoiron/sqlx/sqlx_context.go:128: cannot use db (type *DB) as type ExtContext in argument to NamedQueryContext:
	*DB does not implement ExtContext (missing ExecContext method)
jmoiron/sqlx/sqlx_context.go:134: cannot use db (type *DB) as type ExtContext in argument to NamedExecContext:
	*DB does not implement ExtContext (missing ExecContext method)
jmoiron/sqlx/sqlx_context.go:182: undefined: sql.TxOptions
jmoiron/sqlx/sqlx_context.go:203: undefined: sql.TxOptions
jmoiron/sqlx/sqlx_context.go:134: too many errors
github.com/markbates/pop/columns
github.com/mattn/anko/ast
github.com/daviddengcn/go-colortext
github.com/mattn/anko/parser
gopkg.in/yaml.v2
github.com/shurcooL/github_flavored_markdown
github.com/gobuffalo/buffalo/buffalo/cmd/destroy
github.com/gobuffalo/buffalo/buffalo/cmd/generate
github.com/gobuffalo/plush
github.com/gobuffalo/buffalo/plugins
github.com/markbates/deplist
github.com/markbates/grift/grift
github.com/mattn/anko/vm
github.com/markbates/grift/cmd
github.com/gobuffalo/buffalo/buffalo/cmd/build
github.com/fsnotify/fsnotify
github.com/markbates/sigtx
github.com/markbates/refresh/refresh
github.com/mattn/anko/builtins/encoding/json
github.com/mattn/anko/builtins/flag
github.com/mattn/anko/builtins/errors
github.com/mattn/anko/builtins/fmt
github.com/mattn/anko/builtins/github.com/daviddengcn/go-colortext
github.com/mattn/anko/builtins/io
github.com/mattn/anko/builtins/io/ioutil
github.com/mattn/anko/builtins/math
github.com/mattn/anko/builtins/math/big
github.com/mattn/anko/builtins/math/rand
github.com/mattn/anko/builtins/net
github.com/mattn/anko/builtins/net/http
github.com/mattn/anko/builtins/net/url
github.com/mattn/anko/builtins/os
github.com/mattn/anko/builtins/os/exec
github.com/mattn/anko/builtins/os/signal
github.com/mattn/anko/builtins/path
github.com/mattn/anko/builtins/path/filepath
github.com/mattn/anko/builtins/regexp
github.com/mattn/anko/builtins/runtime
github.com/mattn/anko/builtins/sort
github.com/mattn/anko/builtins/strconv
# github.com/mattn/anko/builtins/sort
mattn/anko/builtins/sort/sort_go18.go:13: undefined: sort.Slice
github.com/mattn/anko/builtins/strings
github.com/mattn/anko/builtins/time
-@--GE72-2QF:~/Documents/projects/src/github.com$ go get -u -v -tags nosqlite github.com/gobuffalo/buffalo/buffalo
*github.com/gobuffalo/buffalo (download)
github.com/fatih/color (download)
github.com/gobuffalo/envy (download)
github.com/joho/godotenv (download)
github.com/mitchellh/go-homedir (download)
github.com/gobuffalo/makr (download)
github.com/markbates/inflect (download)
github.com/pkg/errors (download)
github.com/sirupsen/logrus (download)
Fetching https://golang.org/x/crypto/ssh/terminal?go-get=1
Parsing meta tags from https://golang.org/x/crypto/ssh/terminal?go-get=1 (status code 200)
get "golang.org/x/crypto/ssh/terminal": found meta tag main.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/ssh/terminal?go-get=1
get "golang.org/x/crypto/ssh/terminal": verifying non-authoritative meta tag
Fetching https://golang.org/x/crypto?go-get=1
Parsing meta tags from https://golang.org/x/crypto?go-get=1 (status code 200)
golang.org/x/crypto (download)
Fetching https://golang.org/x/sys/unix?go-get=1
Parsing meta tags from https://golang.org/x/sys/unix?go-get=1 (status code 200)
get "golang.org/x/sys/unix": found meta tag main.metaImport{Prefix:"golang.org/x/sys", VCS:"git", RepoRoot:"https://go.googlesource.com/sys"} at https://golang.org/x/sys/unix?go-get=1
get "golang.org/x/sys/unix": verifying non-authoritative meta tag
Fetching https://golang.org/x/sys?go-get=1
Parsing meta tags from https://golang.org/x/sys?go-get=1 (status code 200)
golang.org/x/sys (download)
github.com/gobuffalo/packr (download)
Fetching https://golang.org/x/sync/errgroup?go-get=1
Parsing meta tags from https://golang.org/x/sync/errgroup?go-get=1 (status code 200)
get "golang.org/x/sync/errgroup": found meta tag main.metaImport{Prefix:"golang.org/x/sync", VCS:"git", RepoRoot:"https://go.googlesource.com/sync"} at https://golang.org/x/sync/errgroup?go-get=1
get "golang.org/x/sync/errgroup": verifying non-authoritative meta tag
Fetching https://golang.org/x/sync?go-get=1
Parsing meta tags from https://golang.org/x/sync?go-get=1 (status code 200)
golang.org/x/sync (download)
Fetching https://golang.org/x/net/context?go-get=1
Parsing meta tags from https://golang.org/x/net/context?go-get=1 (status code 200)
get "golang.org/x/net/context": found meta tag main.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/context?go-get=1
get "golang.org/x/net/context": verifying non-authoritative meta tag
Fetching https://golang.org/x/net?go-get=1
Parsing meta tags from https://golang.org/x/net?go-get=1 (status code 200)
golang.org/x/net (download)
github.com/gobuffalo/plush (download)
github.com/gobuffalo/tags (download)
github.com/fatih/structs (download)
github.com/markbates/validate (download)
github.com/markbates/going (download)
github.com/satori/go.uuid (download)
github.com/serenize/snaker (download)
github.com/russross/blackfriday (download)
github.com/shurcooL/github_flavored_markdown (download)
github.com/microcosm-cc/bluemonday (download)
Fetching https://golang.org/x/net/html?go-get=1
Parsing meta tags from https://golang.org/x/net/html?go-get=1 (status code 200)
get "golang.org/x/net/html": found meta tag main.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/html?go-get=1
get "golang.org/x/net/html": verifying non-authoritative meta tag
Fetching https://golang.org/x/net/html/atom?go-get=1
Parsing meta tags from https://golang.org/x/net/html/atom?go-get=1 (status code 200)
get "golang.org/x/net/html/atom": found meta tag main.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/html/atom?go-get=1
get "golang.org/x/net/html/atom": verifying non-authoritative meta tag
github.com/shurcooL/highlight_diff (download)
github.com/sergi/go-diff (download)
github.com/sourcegraph/annotate (download)
github.com/sourcegraph/syntaxhighlight (download)
github.com/shurcooL/highlight_go (download)
github.com/shurcooL/octiconssvg (download)
github.com/shurcooL/sanitized_anchor_name (download)
github.com/spf13/cobra (download)
github.com/spf13/pflag (download)
github.com/markbates/pop (download)
github.com/cockroachdb/cockroach-go (download)
github.com/lib/pq (download)
github.com/go-sql-driver/mysql (download)
github.com/jmoiron/sqlx (download)
github.com/mattn/anko (download)
github.com/daviddengcn/go-colortext (download)
Fetching https://gopkg.in/yaml.v2?go-get=1
Parsing meta tags from https://gopkg.in/yaml.v2?go-get=1 (status code 200)
get "gopkg.in/yaml.v2": found meta tag main.metaImport{Prefix:"gopkg.in/yaml.v2", VCS:"git", RepoRoot:"https://gopkg.in/yaml.v2"} at https://gopkg.in/yaml.v2?go-get=1
gopkg.in/yaml.v2 (download)
github.com/markbates/deplist (download)
github.com/markbates/grift (download)
github.com/markbates/refresh (download)
github.com/fsnotify/fsnotify (download)
github.com/markbates/sigtx (download)
github.com/fatih/color/vendor/github.com/mattn/go-colorable
github.com/fatih/color/vendor/github.com/mattn/go-isatty
github.com/joho/godotenv
github.com/mitchellh/go-homedir
github.com/gobuffalo/buffalo/generators/assets
github.com/fatih/color
github.com/pkg/errors
golang.org/x/sys/unix
github.com/gobuffalo/envy
golang.org/x/net/context
golang.org/x/sync/errgroup
github.com/gobuffalo/plush/token
github.com/gobuffalo/buffalo/generators
github.com/markbates/inflect
github.com/gobuffalo/packr
github.com/gobuffalo/packr/builder
github.com/gobuffalo/plush/ast
github.com/gobuffalo/plush/lexer
github.com/fatih/structs
github.com/gobuffalo/plush/parser
github.com/gobuffalo/tags
golang.org/x/crypto/ssh/terminal
github.com/markbates/going/wait
github.com/markbates/validate
github.com/satori/go.uuid
github.com/markbates/going/defaults
github.com/serenize/snaker
github.com/sirupsen/logrus
github.com/russross/blackfriday
github.com/markbates/validate/validators
golang.org/x/net/html/atom
golang.org/x/net/html
github.com/sergi/go-diff/diffmatchpatch
github.com/sourcegraph/annotate
github.com/shurcooL/sanitized_anchor_name
github.com/sourcegraph/syntaxhighlight
github.com/spf13/pflag
github.com/gobuffalo/makr
github.com/gobuffalo/buffalo/meta
github.com/gobuffalo/tags/form
github.com/gobuffalo/buffalo/generators/assets/standard
github.com/microcosm-cc/bluemonday
github.com/gobuffalo/buffalo/generators/assets/webpack
github.com/shurcooL/highlight_diff
github.com/shurcooL/highlight_go
github.com/gobuffalo/tags/form/bootstrap
github.com/shurcooL/octiconssvg
github.com/gobuffalo/buffalo/generators/action
github.com/gobuffalo/buffalo/generators/docker
github.com/gobuffalo/buffalo/generators/grift
github.com/gobuffalo/buffalo/generators/mail
github.com/gobuffalo/buffalo/generators/resource
github.com/gobuffalo/buffalo/generators/refresh
github.com/spf13/cobra
github.com/lib/pq/oid
github.com/lib/pq
github.com/go-sql-driver/mysql
# github.com/go-sql-driver/mysql
../../src/github.com/go-sql-driver/mysql/connection_go18.go:42: undefined: driver.TxOptions
../../src/github.com/go-sql-driver/mysql/connection_go18.go:48: undefined: sql.IsolationLevel
../../src/github.com/go-sql-driver/mysql/connection_go18.go:48: undefined: sql.LevelDefault
../../src/github.com/go-sql-driver/mysql/connection_go18.go:62: undefined: driver.NamedValue
../../src/github.com/go-sql-driver/mysql/connection_go18.go:81: undefined: driver.NamedValue
../../src/github.com/go-sql-driver/mysql/connection_go18.go:115: undefined: driver.NamedValue
../../src/github.com/go-sql-driver/mysql/connection_go18.go:134: undefined: driver.NamedValue
../../src/github.com/go-sql-driver/mysql/connection_go18.go:199: undefined: driver.NamedValue
../../src/github.com/go-sql-driver/mysql/utils_go18.go:24: undefined: driver.NamedValue
../../src/github.com/go-sql-driver/mysql/utils_go18.go:36: undefined: driver.IsolationLevel
../../src/github.com/go-sql-driver/mysql/connection_go18.go:48: too many errors
github.com/jmoiron/sqlx/reflectx
# github.com/lib/pq
../../src/github.com/lib/pq/conn_go18.go:15: undefined: driver.NamedValue
../../src/github.com/lib/pq/conn_go18.go:33: undefined: driver.NamedValue
../../src/github.com/lib/pq/conn_go18.go:47: undefined: driver.TxOptions
../../src/github.com/lib/pq/conn_go18.go:50: undefined: sql.IsolationLevel
../../src/github.com/lib/pq/conn_go18.go:51: undefined: sql.LevelDefault
../../src/github.com/lib/pq/conn_go18.go:53: undefined: sql.LevelReadUncommitted
../../src/github.com/lib/pq/conn_go18.go:55: undefined: sql.LevelReadCommitted
../../src/github.com/lib/pq/conn_go18.go:57: undefined: sql.LevelRepeatableRead
../../src/github.com/lib/pq/conn_go18.go:59: undefined: sql.LevelSerializable
github.com/markbates/going/randx
github.com/markbates/pop/columns
github.com/jmoiron/sqlx
github.com/shurcooL/github_flavored_markdown
github.com/mattn/anko/ast
github.com/gobuffalo/buffalo/buffalo/cmd/destroy
# github.com/jmoiron/sqlx
../../src/github.com/jmoiron/sqlx/named_context.go:41: n.Stmt.ExecContext undefined (type *Stmt has no field or method ExecContext)
../../src/github.com/jmoiron/sqlx/named_context.go:41: not enough arguments to return
../../src/github.com/jmoiron/sqlx/named_context.go:51: n.Stmt.QueryContext undefined (type *Stmt has no field or method QueryContext)
../../src/github.com/jmoiron/sqlx/named_context.go:51: not enough arguments to return
../../src/github.com/jmoiron/sqlx/sqlx_context.go:20: db.PingContext undefined (type *DB has no field or method PingContext)
../../src/github.com/jmoiron/sqlx/sqlx_context.go:122: cannot use db (type *DB) as type namedPreparerContext in argument to prepareNamedContext:
	*DB does not implement namedPreparerContext (missing PrepareContext method)
../../src/github.com/jmoiron/sqlx/sqlx_context.go:128: cannot use db (type *DB) as type ExtContext in argument to NamedQueryContext:
	*DB does not implement ExtContext (missing ExecContext method)
../../src/github.com/jmoiron/sqlx/sqlx_context.go:134: cannot use db (type *DB) as type ExtContext in argument to NamedExecContext:
	*DB does not implement ExtContext (missing ExecContext method)
../../src/github.com/jmoiron/sqlx/sqlx_context.go:182: undefined: sql.TxOptions
../../src/github.com/jmoiron/sqlx/sqlx_context.go:203: undefined: sql.TxOptions
../../src/github.com/jmoiron/sqlx/sqlx_context.go:134: too many errors
github.com/gobuffalo/buffalo/buffalo/cmd/generate
github.com/daviddengcn/go-colortext
gopkg.in/yaml.v2
github.com/gobuffalo/buffalo/plugins
github.com/mattn/anko/parser
github.com/gobuffalo/plush
github.com/markbates/deplist
github.com/markbates/grift/grift
github.com/markbates/grift/cmd
github.com/fsnotify/fsnotify
github.com/gobuffalo/buffalo/buffalo/cmd/build
github.com/markbates/sigtx
github.com/mattn/anko/vm
github.com/markbates/refresh/refresh
github.com/mattn/anko/builtins/encoding/json
github.com/mattn/anko/builtins/errors
github.com/mattn/anko/builtins/flag
github.com/mattn/anko/builtins/fmt
github.com/mattn/anko/builtins/github.com/daviddengcn/go-colortext
github.com/mattn/anko/builtins/io
github.com/mattn/anko/builtins/io/ioutil
github.com/mattn/anko/builtins/math
github.com/mattn/anko/builtins/math/big
github.com/mattn/anko/builtins/math/rand
github.com/mattn/anko/builtins/net
github.com/mattn/anko/builtins/net/http
github.com/mattn/anko/builtins/net/url
github.com/mattn/anko/builtins/os
github.com/mattn/anko/builtins/os/exec
github.com/mattn/anko/builtins/os/signal
github.com/mattn/anko/builtins/path
github.com/mattn/anko/builtins/path/filepath
github.com/mattn/anko/builtins/regexp
github.com/mattn/anko/builtins/runtime
github.com/mattn/anko/builtins/sort
github.com/mattn/anko/builtins/strconv
github.com/mattn/anko/builtins/strings
github.com/mattn/anko/builtins/time
# github.com/mattn/anko/builtins/sort
../../src/github.com/mattn/anko/builtins/sort/sort_go18.go:13: undefined: sort.Slice
*

**Comment by stanislas-m** [comment]
There is a Slack channel, with friendly people. :) https://gobuffalo.io/docs/slack

Check the buffalo binary is is the `$GOPATH/bin`:

```bash
$ ls -l $GOPATH/bin/buffalo 
-rwxr-xr-x 1 michalakst michalakst 20385456 févr.  2 17:36 /home/michalakst/Documents/Dev/go/bin/buffalo
```

If not, you can install it with the `go install` command:

```bash
go install -v github.com/gobuffalo/buffalo/buffalo
```

Then ensure you have `$GOPATH/bin` in your `$PATH`.

**Comment by hubyhuby** [comment]
Hi @stanislas-m,
Thanks a lot for your help!
Note that I could install the revel framework with the go get without any troubel and get the revel command and my first project to run. So I supposed that my go install was quite fine.

Yes the buffalo bin wasn t there (the revel is there and others)
I tried again but it didn t work either.
I finally logged in root on my ubuntu 17.10 (sudo -i) and did the go get , suddenly everything installed fine.
I had to set all the environment variable of go for root, but it do work . yeah!

Bottom line on my system : I can t install using regular user permissions and can using root.
I have a feeling it is related to sqlite3 and pq libraries...might be with the gcc...

**Comment by stanislas-m** [comment]
@hubyhuby I guess it's a permission issue on your side. Make sure the permissions are correct, and the $GOPATH is set in a place you have the rights to write.

If the gcc works with the root account, it should work with the non-root one. I'm using Ubuntu 17.04, with a normal account, and it works fine.
